medical-portal/src/API/RSBC.DMF.MedicalPortal.API/ViewModels/Flag.cs

```diff
@@ -0,0 +1,20 @@
+namespace RSBC.DMF.MedicalPortal.API.ViewModels;
+
+/// <summary>
+/// Enum for Flag Type Option Set
+/// </summary>
+public enum FlagTypeOptionSet
```

Comment:
cms-adapter should be hiding anything Dynamics specific, but I also find it useful to know these values outside of cms-adapter. I would add a comment here, like the one I added for SubmittalStatus enum
`// NOTE that the dynamic values here should only be used in cms-adapter only and they are only here for reference and ease of mapping values`